### ISSUE TITLE
React Native 0.56 Support

### ISSUE THREAD

**Issue by DaniAkash** [ak1394/react-native-tts#91]
In React Native 0.56 
Android projects are compiled by SDK level 26 as default 

https://github.com/react-native-community/react-native-releases/blob/master/CHANGELOG.md#android-projects-are-now-compiled-using-the-android-26-sdk

This causes the following issue with this plugin:

```
* What went wrong:
A problem occurred configuring project ':app'.
> Could not resolve all dependencies for configuration ':app:_debugApk'.
   > A problem occurred configuring project ':react-native-tts'.
      > The SDK Build Tools revision (23.0.1) is too low for project ':react-native-tts'. Minimum required is 25.0.0
```

It'd be helpful to know if any work arounds are available until this plugin supports 0.56

**Comment by ak1394** [comment]
No workaround I'm aware of at the moment, I plan to look into adding support for 0.56 some time this week.

**Comment by ak1394** [comment]
Published v 2.0.0 with 0.56 support.
